src/scripts/plugin.js

```diff
@@ -9,15 +9,15 @@ const thebelabConfig = {
 const languageDictionary = {
   'Python 3': ['python'],
   Julia: ['julia'],
-  R: ['R'],
+  r: ['R'],
   Octave: ['octave'],
-  SageMath: ['SageMath'],
+  sagemath: ['SageMath'],
```

Comment:
It seems like Code Mirror does not support syntax highlight for SageMath at all. It will not work if you press `create HTML`. See Code Mirror's [support modes](https://codemirror.net/mode/) for reference. I think we should decide on how to deal with this. I am ok with not having support. But maybe we can try to use python's syntax highlighting.

Reply:
are your? The colors do appear.

![image](https://user-images.githubusercontent.com/53124295/74317209-d0329900-4d2f-11ea-82d1-031085ecc8e9.png)

Reply:
As mentioned, try pressing `create HTML`.

Reply:
It's fine to use python highlighting for sage.